core/src/main/java/org/openstack4j/api/Apis.java

```diff
@@ -231,4 +232,13 @@ private static APIProvider initializeProvider() {
         p.initialize();
         return p;
     }
+    
+    /**
+     * Gets the Server events services API
+     *
+     * @return the event Services
+     */
+    public static EventServices getEventServices() {
+        return get(EventServices.class);
```

Comment:
Provided this is scoped to compute, it is better to have this only in `ComputeService`. So it will be `Apis.getComputeServices()...`